### PR TITLE
Remove console usage

### PR DESCRIPTION
**Pull request by jourdain**

### CODE REVIEW

**Comment by jourdain** [comment]
@martinken do you mind testing it on edge?

File: Sources/macro.js [hunk 1]
```diff
@@ -7,6 +7,52 @@ export function getCurrentGlobalMTime() {
...
+// Loggins function calls
+// ----------------------------------------------------------------------------
+
+const fakeConsole = {};
+function noOp() {}
+
+const consoleMethods = ['log', 'debug', 'info', 'warn', 'error', 'time', 'timeEnd', 'group', 'groupEnd'];
+consoleMethods.forEach(methodName => (fakeConsole[methodName] = noOp));
+
+global.console = window.console || fakeConsole;
```
Comment (aronhelser) [inline Sources/macro.js]: I feel like we need to do this for ParaViewWeb (and ERS) - is there a good place?

Comment (jourdain) [inline Sources/macro.js]: Agree but don't know where yet...

Comment (jourdain) [inline Sources/macro.js]: Maybe in Common/Core/Console

Comment (jourdain) [inline Sources/macro.js]: Then we just do a `import 'paraviewweb/src/Common/Core/Console' somewhere...

Comment (aronhelser) [inline Sources/macro.js]: I don't get a crash on Edge in the ERS project when doing console.log - maybe babel or something is already doing a polyfill for us?

Comment (aronhelser) [inline Sources/macro.js]: Yup, seems to be happening in autobahn\lib\polyfill\console.js

Comment (jourdain) [inline Sources/macro.js]: Good to know but we should have our own as we don't always have autobahn.

**Pull request merged**